auth/server/src/main/java/com/seagates3/authserver/AuthServer.java

```diff
@@ -187,7 +187,16 @@ static void main(String[] args) throws InterruptedException,
             serverChannels.add(serverChannel);
             logger.info("Auth server is listening on HTTPS port " + httpsPort);
           }
-
+          // listner code starts here
+          AuthServerConfigChangeListner listner =
+              new AuthServerConfigChangeListner(
+                  AuthServerConstants.RESOURCE_DIR + "/authserver.properties");
+          try {
+            new Thread(listner).start();
```

Comment:
I hope there is no need to detach this thread

Reply:
No need to detach because if we detach thread then it will not be able to listen to events happenings on config file.
Hence thread will be running till authserver is up and running.